Makefile

```diff
@@ -87,6 +87,11 @@ lint: generate vanity-import-check pprof-execs
 	$(MAKE) lint -C support/ebpf
 	docker run --rm -t -v $$(pwd):/app -w /app golangci/golangci-lint:$(GOLANGCI_LINT_VERSION) sh -c "golangci-lint version && golangci-lint config verify && golangci-lint run --max-issues-per-linter -1 --max-same-issues -1"
 
+format: format-ebpf
+	gofmt -w -s ./
+	go tool goimports -w ./
+	go tool gofumpt -w ./
```

Comment:
Let's drop `gofumpt` as its the culprit for generating the unwanted clutter. We can talk about changing/adding it in a later PR when the 3-6 large PRs that need large rebasing are merged. I personally dislike the way it changing the function signatures. But there are also some good changes. Can the function signature changing be disabled?